src/Form/FilterRoleType.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace AppBundle\Form;
+
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+class FilterRoleType extends AbstractType
+{
+    public const ROLES = [
+        'adherent_message.filter.role.committeeHosts' => 'CommitteeHosts',
+        'adherent_message.filter.role.committeeSupervisors' => 'CommitteeSupervisors',
+        'adherent_message.filter.role.citizenProjectHosts' => 'CitizenProjectHosts',
+    ];
```

Comment:
what about keeping this array "clean" with only values,
and add this in the `setDefaults` method:
```
'choice_label' => function (string $choice) {
    return "adherent_message.filter.role.$choice";
},
```

Reply:
done